### PR TITLE
Added support for selecting all entities of an entity kind from an entity group

### PR DESCRIPTION
**Pull request by wesokes**
This adds query support for filtering groups that contain a null entity with a sub_entity_kind (all entities of a specific kind).

@Wilduck @jmcriffey @wesleykendall please review

### CODE REVIEW

File: entity/tests/model_tests.py [hunk 1]
```diff
@@ -792,7 +803,7 @@ def test_filters_groups(self):
           entity=e, sub_entity_kind=self.kind1)
         G(EntityGroupMembership, entity_group=self.group,
           entity=e, sub_entity_kind=None)
-        G(EntityGroupMembership, entity_gropu=other_group,
+        G(EntityGroupMembership, entity_group=other_group,
```
Comment (Wilduck) [inline entity/tests/model_tests.py]: Nice catch. :flushed:

**Comment by Wilduck** [comment]
This looks great @wesokes

**Comment by jmcriffey** [comment]
LGTM, thanks for digging into this @wesokes.

File: entity/models.py [hunk 2]
```diff
@@ -450,5 +449,5 @@ class EntityGroupMembership(models.Model):
     accessed through the EntityGroup api.
     """
     entity_group = models.ForeignKey(EntityGroup)
-    entity = models.ForeignKey(Entity)
+    entity = models.ForeignKey(Entity, null=True)
```
Comment (wesleykendall) [inline entity/models.py]: @wesokes what is the meaning of Null here?

**Comment by wesleykendall** [comment]
@wesokes one question

File: entity/models.py (hunk 2 above)
Comment (wesokes) [inline entity/models.py]: if entity is null, it means "all entities from the specified sub entity type"

Comment (wesokes) [inline entity/models.py]: it is so we can specify all accounts within a team or other relationships similar to that

Comment (wesokes) [inline entity/models.py]: sorry i just type wrong info. when this is null, it just means all entities of the sub entity kind. so the ui will let you select "Account" or another entity kind, then you can select "All", which just means every active entity of that kind.

Comment (wesleykendall) [inline entity/models.py]: I see, can you comment that?

**Comment by wesleykendall** [comment]
@wesokes feel free to merge after that comment

**Pull request merged**